nixos/modules/security/acme.nix

```diff
@@ -418,29 +420,29 @@ let
 
   certConfigs = mapAttrs certToConfig cfg.certs;
 
-  certOpts = { name, ... }: {
-    options = {
-      # user option has been removed
-      user = mkOption {
-        visible = false;
-        default = "_mkRemovedOptionModule";
-      };
-
-      # allowKeysForGroup option has been removed
-      allowKeysForGroup = mkOption {
-        visible = false;
-        default = "_mkRemovedOptionModule";
+  # These options can be specified within
+  # security.acme or security.acme.certs.<name>
+  inheritableOpts =
+    { inheritDefaults ? false, defaults ? null }: {
+      validMinDays = mkOption {
+        type = types.int;
+        default = if inheritDefaults then defaults.validMinDays else 30;
```

Comment:
please add a defaulyText explaining what happens here, otherwise it'll show up as just `30` in the manual. (also for other options, not tagging them all)

Reply:
Done now. The format is `"Inherit from security.acme.defaults, otherwise ${val}"`